app/src/citations.py

```diff
@@ -26,7 +26,24 @@ def create_citation(self, chunk: Chunk, subsection: str) -> Subsection:
 
 
 def default_chunk_splitter(chunk: Chunk) -> list[str]:
```

Comment:
`default_chunk_splitter` is used by `split_into_subsections()` which is used in `BaseEngine`.